packages/pulp-docker/pulp-docker.spec

```diff
@@ -3,13 +3,13 @@
 
 # ---- archive related macros ----
 
-%define git_tag 3.1.2
+%define git_tag 3.1.4
 %define srcname pulp_docker
 
 
 Name: pulp-docker
 Version: 3.1.4
```

Comment:
Should this be `%{git_tag}` to ensure they don't go out of sync again?

Reply:
Yes and no....  To prevent premature tagging of the repo, we sometimes set 'git_tag' to the branch we want to build from, for our test builds that we throw away.  We could set git_tag to %{version} though, and that should ensure it going forward

Reply:
updated to use git_tag %{version}